script.js

```diff
@@ -1,77 +1,81 @@
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['Ruby', 'CSS', 'JavaScript', 'Bootstrap'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 2,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 3,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 4,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 5,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+//   {
+//     id: 6,
+//     name: 'Project name gors here',
+//     paragraph1: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     paragraph2: 'loremLorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia',
+//     'mobile image': 'images.svg/Rectangle 34.svg',
+//     'desktop image': 'images.svg/Rectangle 34 (1).svg',
+//     technologies: ['HTML/CSS', 'RUBY ON RAILS', 'JAVASCRIPT'],
+//     liveVersion: '#',
+//     source: '#',
+//   },
+// ];
```

Comment:
- [ ] Kindly do not leave commented code in your code base, they make your project look messy.